infra/gcb/build_project.py

```diff
@@ -206,8 +215,8 @@ def get_build_steps(project_dir):
                     # `cd /src && cd {workdir}` (where {workdir} is parsed from
                     # the Dockerfile). Container Builder overrides our workdir
                     # so we need to add this step to set it back.
-                    ('rm -r /out && cd /src && cd {workdir} && mkdir -p {out} && '
```

Comment:
why this change?

Reply:
linter complained, looks like some of the previous formatting changes increased the indentation here and the line became too long